pkg/operators/kubemanager/kubemanager.go

```diff
@@ -244,81 +244,85 @@ func (m *KubeManagerInstance) PreGadgetRun() error {
 		m.params.Get(ParamAllNamespaces).AsBool(),
 	)
 
-	if setter, ok := m.gadgetInstance.(MountNsMapSetter); ok {
-		err := m.manager.gadgetTracerManager.AddTracer(m.id, containerSelector)
-		if err != nil {
-			return fmt.Errorf("adding tracer: %w", err)
-		}
+	if m.gadgetInstance != 0 {
```

Comment:
Why 0? The localmanager operator is using `nil`. Also same comment about using an early return.